pom.xml

```diff
@@ -53,7 +52,7 @@
         <metrics-jdbi3.version>4.1.19</metrics-jdbi3.version>
         <metrics-jetty9.version>4.1.19</metrics-jetty9.version>
         <mongo-java-driver.version>3.12.8</mongo-java-driver.version>
-        <zookeeper.version>3.4.14</zookeeper.version>
+        <zookeeper.version>3.7.0</zookeeper.version>
```

Comment:
OK, assuming we verify Curator 5.1.0 fully works with this (Curator 5.1.0 depends on ZK 3.6.x)

Reply:
Verified that dropwizard-curator and dropwizard-leader-latch both work with curator 5.1.0 and zk 3.7.0